### PR TITLE
Issue/gh 284

### PR DESCRIPTION
**Pull request by mibhumi**
Reference of issue: https://github.com/rtCamp/rtMedia/issues/284

### CODE REVIEW

File: app/assets/css/sass/_rtm.scss [hunk 1]
```diff
@@ -197,6 +197,15 @@ button#rtmedia-add-media-button-post-update {
 	.rtm-field-wrap {
 		margin-bottom: 20px;
 	}
+
+	.rtmedia-editor-buttons {
+		height: 25px;
+
+		input[type= submit] {
```
Comment (dharmin) [inline app/assets/css/sass/_rtm.scss]: Remove extra space between after equal sign.

**Comment by pooja-muchandikar** [comment]
@kapilpaul 

The icon change is working.

PR change screenshot
Screenshot 2021-10-01 at 6 37 01 PM

---

Existing screenshot
Screenshot 2021-10-01 at 6 37 27 PM

---

The other change is not understandable.

**Comment by kapilpaul** [comment]
@pooja-muchandikar we can just pick the commit which changes are visible. the main issue is closed.

**Comment by pooja-muchandikar** [comment]
@kapilpaul 

Yes that works we can pick only one commit

**Comment by kapilpaul** [comment]
cherry picked [one commit](https://github.com/rtCamp/rtMedia/pull/1336/commits/734c557b85e727183042840e5f29a82d7bfc7f7f).

**Pull request closed without merging**